pkg/utils/logger/logger.go

```diff
@@ -47,22 +47,17 @@ type Logger interface {
 
 // Get returns an default instance of the zap logger
 func Get() Logger {
-	var logf = &structuredLogger{}
```

Comment:
Why are we removing this structuredLogger reference? and isEmpty() call?

Reply:
Here logf will always be empty, hence it was removed.

But I am thinking should we instead implement isEmpty() for the global variable and assign the local variable to global variable if the global is empty. Just to keep the code readable.

Reply:
Existing logic was 
```
var logf = &structuredLogger{}
	if logf.isEmpty() {
```	

```
func (logf *structuredLogger) isEmpty() bool {
	return logf.zapLogger == nil
}
```

Through log messages, it was confirmed that each package was obtaining it's own logger instance:
Test execution logs (prior to the change)
```
"level":"info","ts":"2022-07-07T23:02:41.525Z","caller":"eniconfig/eniconfig.go:61","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.526Z","caller":"networkutils/network.go:128","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.531Z","caller":"awssession/session.go:39","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.531Z","caller":"ec2wrapper/ec2wrapper.go:21","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.537Z","caller":"k8sapi/k8sutils.go:23","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.537Z","caller":"eventrecorder/eventrecorder.go:33","msg":"******** : Initialized new logger"}
{"level":"info","ts":"2022-07-07T23:02:41.538Z","caller":"awsutils/awsutils.go:81","msg":"******** : Initialized new logger"}
```

Thus, the extra method was removed and nil check was moved in-line, keeping the code simple and easy to read.

Reply:
Understood, and that was the only place it was used, so helps to remove it.

Thank you.